gspread/models.py

```diff
@@ -642,7 +642,7 @@ def get_all_records(
             for row in data[idx + 1:]
         ]
 
-        return [dict(zip(keys, row)) for row in values]
+        return [OrderedDict(dict(zip(keys, row))) for row in values]
```

Comment:
Wouldn't passing `dict` to `OrderedDict` defeat the purpose in this case?

Python 2.7.15:
```python
>>> from collections import OrderedDict
>>> dict([('c', 1), ('a', 2), ('b', 3)]).keys()
['a', 'c', 'b']
>>> OrderedDict(dict([('c', 1), ('a', 2), ('b', 3)])).keys()
['a', 'c', 'b']
>>> OrderedDict([('c', 1), ('a', 2), ('b', 3)]).keys()
['c', 'a', 'b']
```